operations/views.py

```diff
@@ -44,6 +44,20 @@ class ParticipantViewSet(viewsets.ModelViewSet):
     serializer_class = ParticipantSerializer
 
 
+class ParticipantSearchList(generics.ListAPIView):
```

Comment:
You don't need any of this really. You can just add `filter_fields = ('msisdn',)` to `ParticipantViewSet` and then `api/v1/participants/?msisdn=%2B27820010001` will work